cmd/bootstrap/PROVIDED_CERTIFICATES.md

```diff
@@ -0,0 +1,150 @@
+
+After a site bundle has been produced, it contains the whole site definition that can be installed
+at a remote location. Thus, all certificates and static links are already part of the bundle and no
+new certificate is signed at the moment a bundle is installed.
+
+## Providing your own certificates
+
+You can provide your own certificates to be used by Skupper for site linking.
+Depending on your goals, some certificates should be supplied at certain phases,
+for example, at the time a local site is being initialized, a bundle is being prepared
+or a bundle is being installed at a remote location.
+
+To understand it better, let's go through the main use cases and review what is the ideal
+phase that a given kind of certificate should be provided.
+
+### Using a custom CA to sign certificates
+
+If you want Skupper to generate client and server certificates signed by a custom CA,
+you will need to provide the respective certificates during:
+
+* Local site initialization time
+* Site bundle preparation time
+
+Certificates are signed by Skupper at the time a local namespace is being initialized
+or a bundle is being produced.
+
+After a site bundle has been produced, it already contains the whole definition and no
+new certificate is expected to be signed.
+
+### Using custom server and client certificates
+
+If you want a new local site to use your custom server and client certificates, you can
+provide them at any time, for example:
+
+* Local site initialization time
+* Site bundle preparation time
+* Site bundle installation time
+
+During "Local site initialization time" or "Site bundle preparation time", Skupper will detect
+that a server certificate has been provided and will inspect it to determine its subject
+alternative names and based on that, a set of static links will be created, allowing those
+links to be distributed to target sites accordingly. For the static links to remain valid,
+the expected client certificate must also be provided.
+
+If a server certificate is provided at "Site bundle installation time", Skupper will also try to
+determine its subject alternative names using the `openssl` binary (only if available) and it will
+use it to generate the static links for the bundle installation. This way, the set of static links
+available for an installed bundle will be valid for all expected target hostnames and IP addresses
+defined through the server certificate.
+
+Again, if a server certificate is provided, the respective client certificate is also expected
+so that the static links have valid client credentials.
+
+## Examples
+
+## Provide your own skupper-site-ca
+
+If you want Skupper to use your own CA certificates to generate and sign server and client
+certificates used for site linking, you can simply create the following structure under
+the namespace home of your choice, for example:
+
+```shell
+${HOME}/.local/share/skupper/namespaces/default/input/certificates/
+└── ca
+    └── skupper-site-ca
```

Comment:
This mostly answers my question above, but perhaps worth clarifying that the name of the directory is important here?

Reply:
Agreed. The directory name is actually what identifies the certificate.
I will add that information earlier in the document to make it clear.